<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InstallForm</class>
 <widget class="QMainWindow" name="InstallForm">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>438</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>1280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Portfolio Synode 0.7</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="jservLines">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>135</y>
      <width>521</width>
      <height>61</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>8000</width>
      <height>400</height>
     </size>
    </property>
   </widget>
   <widget class="QGroupBox" name="gboxRegistry">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>941</width>
      <height>131</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>  Synchronization Domain Identity - Market TEST</string>
    </property>
    <widget class="QLabel" name="lbCommunity">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>100</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Community:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDomain">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>96</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string extracomment="Domain wide node ID."/>
     </property>
     <property name="text">
      <string>zsu</string>
     </property>
     <property name="maxLength">
      <number>12</number>
     </property>
     <property name="placeholderText">
      <string>12 Bit Market wide ID, must not null</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbRetry">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>31</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirm:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPswd2">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>28</y>
       <width>171</width>
       <height>27</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string extracomment="Domain wide node ID."/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>32</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Confirm Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbAdminId">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>31</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Admin:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPswd">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>28</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string extracomment="Domain wide node ID."/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>32</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Max 32 Chars</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtAdminId">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>28</y>
       <width>131</width>
       <height>27</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string extracomment="Domain wide node ID."/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>12</number>
     </property>
     <property name="placeholderText">
      <string>admin ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbPswd">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>31</y>
       <width>70</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bRegfolder">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>62</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>  &amp;...</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::FolderNew"/>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>22</height>
      </size>
     </property>
     <property name="shortcut">
      <string>Ctrl+D</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbRegfolder">
     <property name="geometry">
      <rect>
       <x>39</x>
       <y>65</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Registry:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtResroot">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>62</y>
       <width>651</width>
       <height>27</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string extracomment="Managed files and data are saved here."/>
     </property>
     <property name="text">
      <string>./registry</string>
     </property>
     <property name="placeholderText">
      <string>Path to the registered resources.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bLoadPeers">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>96</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string extracomment="Test">&amp;Find Peers</string>
     </property>
     <property name="shortcut">
      <string>Ctrl+S</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbHelplink">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>0</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Help</string>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lbDomain">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>100</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Domain:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCommunity">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>96</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string extracomment="Domain wide node ID."/>
     </property>
     <property name="text">
      <string>ura</string>
     </property>
     <property name="maxLength">
      <number>12</number>
     </property>
     <property name="placeholderText">
      <string>12 Bit Market wide ID, must not null</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bCommDropdown">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>100</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::GoDown"/>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>22</height>
      </size>
     </property>
     <property name="shortcut">
      <string>Ctrl+D</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bLogin">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>27</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string extracomment="Test">&amp;Login</string>
     </property>
     <property name="shortcut">
      <string>Ctrl+S</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bSignup">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>27</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string extracomment="Test">Sign &amp;up</string>
     </property>
     <property name="shortcut">
      <string>Ctrl+S</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDomain_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>96</y>
       <width>171</width>
       <height>27</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string extracomment="Domain wide node ID."/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>12</number>
     </property>
     <property name="placeholderText">
      <string>Used in Domain</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbDomain_2">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>100</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Private Token:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bCommDropdown_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>100</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::GoDown"/>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>22</height>
      </size>
     </property>
     <property name="shortcut">
      <string>Ctrl+D</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lbQr">
    <property name="geometry">
     <rect>
      <x>784</x>
      <y>141</y>
      <width>161</width>
      <height>171</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>200</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bValidate">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>360</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string extracomment="Test">&amp;Test Run</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+T</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bWinserv">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>360</y>
      <width>301</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Install as &amp;Windows Service</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+R</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bSetup">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>323</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string extracomment="Test">&amp;Save</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+T</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bPing">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>163</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string extracomment="Test">&amp;Ping</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+T</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtimeout">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>137</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="whatsThis">
     <string extracomment="Global / Domain wide node ID."/>
    </property>
    <property name="inputMask">
     <string>0000</string>
    </property>
    <property name="text">
     <string>12</string>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbTimeout">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>138</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Timeout(s)</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblink">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Portfolio is based on Open Source Projects.</string>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbVolume_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Peer Services</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gboxLocal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>761</width>
      <height>153</height>
     </rect>
    </property>
    <property name="title">
     <string>Local Synode</string>
    </property>
    <widget class="QLabel" name="lbPort_4">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>90</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbJservs">
     <property name="geometry">
      <rect>
       <x>241</x>
       <y>30</y>
       <width>381</width>
       <height>21</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="text">
      <string>seconds synchronize  with the peers.</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbIneach">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>30</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>In every</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPort_proxy">
     <property name="geometry">
      <rect>
       <x>641</x>
       <y>86</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string extracomment="Global / Domain wide node ID."/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>6</number>
     </property>
     <property name="placeholderText">
      <string>E. g. 8964</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtSynode">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>56</y>
       <width>131</width>
       <height>27</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string extracomment="Domain wide node ID."/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>12</number>
     </property>
     <property name="placeholderText">
      <string>12 Bit GUID, must not null</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbVolume">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>122</y>
       <width>51</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Volume</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbPort_2">
     <property name="geometry">
      <rect>
       <x>629</x>
       <y>59</y>
       <width>12</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbSynode">
     <property name="geometry">
      <rect>
       <x>43</x>
       <y>59</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Synode</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbIP">
     <property name="geometry">
      <rect>
       <x>261</x>
       <y>59</y>
       <width>69</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Local IP</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkReverseProxy">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>89</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Via Proxy </string>
     </property>
    </widget>
    <widget class="QLabel" name="lbPort">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>59</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtWebport">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>56</y>
       <width>101</width>
       <height>27</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string extracomment="Global / Domain wide node ID."/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>6</number>
     </property>
     <property name="placeholderText">
      <string>E. g. 8900</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bVolpath">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>122</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&amp;Volume</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::FolderVisiting"/>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="shortcut">
      <string>Ctrl+D</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtIP">
     <property name="geometry">
      <rect>
       <x>321</x>
       <y>56</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string extracomment="Global / Domain wide node ID."/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>39</number>
     </property>
     <property name="placeholderText">
      <string>0.0.0.0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtWebport_proxy">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>86</y>
       <width>101</width>
       <height>27</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string extracomment="Global / Domain wide node ID."/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>6</number>
     </property>
     <property name="placeholderText">
      <string>E. g. 8900</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbPublicIP">
     <property name="geometry">
      <rect>
       <x>255</x>
       <y>90</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Public IP</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtIP_proxy">
     <property name="geometry">
      <rect>
       <x>321</x>
       <y>86</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string extracomment="Global / Domain wide node ID."/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>39</number>
     </property>
     <property name="placeholderText">
      <string>0.0.0.0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPort">
     <property name="geometry">
      <rect>
       <x>641</x>
       <y>56</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string extracomment="Global / Domain wide node ID."/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>6</number>
     </property>
     <property name="placeholderText">
      <string>E. g. 8964</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtSyncIns">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string extracomment="Global / Domain wide node ID."/>
     </property>
     <property name="inputMask">
      <string>000000</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>6</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <property name="placeholderText">
      <string>120.0</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbPort_3">
     <property name="geometry">
      <rect>
       <x>629</x>
       <y>88</y>
       <width>12</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtVolpath">
     <property name="geometry">
      <rect>
       <x>101</x>
       <y>122</y>
       <width>521</width>
       <height>27</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string extracomment="Managed files and data are saved here."/>
     </property>
     <property name="text">
      <string>../vol</string>
     </property>
     <property name="placeholderText">
      <string>Volume is where the data saved at local node.</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPortfolio_0_7">
    <property name="title">
     <string>Portfolio Synode 0.7</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuPortfolio_0_7"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>txtAdminId</tabstop>
  <tabstop>txtPswd</tabstop>
  <tabstop>txtPswd2</tabstop>
  <tabstop>bSignup</tabstop>
  <tabstop>bLogin</tabstop>
  <tabstop>txtResroot</tabstop>
  <tabstop>bRegfolder</tabstop>
  <tabstop>txtCommunity</tabstop>
  <tabstop>bCommDropdown</tabstop>
  <tabstop>txtDomain</tabstop>
  <tabstop>bCommDropdown_2</tabstop>
  <tabstop>txtDomain_2</tabstop>
  <tabstop>bLoadPeers</tabstop>
  <tabstop>jservLines</tabstop>
  <tabstop>txtimeout</tabstop>
  <tabstop>bPing</tabstop>
  <tabstop>txtSyncIns</tabstop>
  <tabstop>txtSynode</tabstop>
  <tabstop>txtIP</tabstop>
  <tabstop>txtWebport</tabstop>
  <tabstop>txtPort</tabstop>
  <tabstop>chkReverseProxy</tabstop>
  <tabstop>txtIP_proxy</tabstop>
  <tabstop>txtWebport_proxy</tabstop>
  <tabstop>txtPort_proxy</tabstop>
  <tabstop>txtVolpath</tabstop>
  <tabstop>bVolpath</tabstop>
  <tabstop>bSetup</tabstop>
  <tabstop>bValidate</tabstop>
  <tabstop>bWinserv</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
